examples/tpu_training-tf.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Help me understand this. </p><p> </p><p>Since the dataset can fit into memory (and therefore the converted TFRecord), <code>model.fit(tf_dataset)</code> works here without errors on Colab TPUs?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>The dataset could fit into memory, but in this example we save it as TFRecord files on the local disk of the VM, and as a result the TPU node cannot read it. If you try running this code on Colab, you get this error:</p><p> </p><p><code>UNIMPLEMENTED: {{function_node __inference_train_function_24666}} File system scheme '[local]' not implemented (file: 'dataset.tfrecords')</code></p>

Reply:
<p>Maybe throw in another comment to explain, this code will run perfectly fine on a CPU or a TPU VM but not on a Colab TPU throwing the error you snippeted (I just invented this term). </p>

Reply:
<p>Done!</p>